src/Tasks/ManifestUtil/ManifestReader.cs

```diff
@@ -138,7 +142,10 @@ public static Manifest ReadManifest(string manifestType, string path, bool prese
             using (Stream s = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.Read))
             {
                 byte[] buffer = new byte[2];
+#pragma warning disable CA2022 // Avoid inexact read with 'Stream.Read' The check of bytes happens later in the code. In case of invalid document the exception is expected later
```

Comment:
Why did you choose to warning-disable these instead of doing it in editorconfig?

Reply:
The only reason was: Turning off the warning globally will affect future contributions and will not report possible error/warnings